### PR TITLE
198 sync false positive

### PR DESCRIPTION
**Pull request by Szarad**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -342,6 +316,35 @@ async def get_redis(user: str):
...
+        else:
+            return []
+
+
+# this information will be taken from the authentication token
+# userId = "karolina.bozek@witty.works"
+userId = ""
+if userId:
+    corporate_false_positive = get_false_positive(userId)
+else:
```
Comment (lsmith77) [inline app/main.py]: feel free to just sent an empty list here as these words were just for Deutsche Bahn

**Review by lsmith77 — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Szarad) [reply app/main.py]: ok, changing

**Review by lsmith77 — APPROVED** [review]

**Comment by Szarad** [comment]
It is ok that after removing this false-positive list, result from tests/test_false_positive/test_api_corporate_false_positives test with input `{
    "text": "Die Bahn ist stark wegen ihrer Führungskräfte!",
    "config": {
        "store_context": false
    }
}`
 is:
`
E                     ?                 -
E                     +     "results": [
E                     +         {
E                     +             "alternatives": [
E                     +                 "stabil",
E                     +                 "mit Erfahrung",
E                     +                 "fachkundig "
E                     +             ],
E                     +             "category": "unconscious_bias",
E                     +             "context": "",
E                     +             "end": 18,
E                     +             "label": "Sprachliche Voreingenommenheit: Agentisch",
E                     +             "reason": "Unbewusst dem männlichen Geschlecht zugeschrieben. Viele werden davon abgestossen.",
E                     +             "solution": "Teamorientierte Formulierung nutzen oder auf Sinnhaftigkeit einzahlend.",
E                     +             "start": 13,
E                     +             "subcategory": "agentic",
E                     +             "text": "stark"
E                     +         }
E                     +     ]
E                       }
`

 I can update this test with changing false_positive list to empty list? Before that change it was `"results": []` (just empty list)

**Comment by lsmith77** [comment]
the test should be adapted to move the old list into redis I guess

File: app/main.py [hunk 2]
```diff
@@ -241,17 +248,33 @@ async def custom_http_exception_handler(request, e):
...
+    fp = FalsePositive(
+        gender_false_positive + corporate_false_positive,
+        false_positive_agentic_const + corporate_false_positive,
+    )
+    return fp
+
+
+# terms_false_positive = gender_false_positive + corporate_false_positive
+# false_positive_agentic += corporate_false_positive
+false_positive = get_false_positive(gender_false_positive, false_positive_agentic_const)
```
Comment (Szarad) [inline app/main.py:363]: I had to change global variables terms_false_positive and false_positive_agentic  and wrapped it up in the function  get_false_positive. It was impossible to test getting these variables from redis as they were loaded during import.  Let's discuss on Tuesday refactoring our code to make test easier.

File: Pipfile.lock [hunk 3]
```diff
@@ -282,30 +282,29 @@
...
-                "sha256:97199a13b772e74cdcdb03760c32109c808aff7cd49c29e9cf4b7754bb725d1d",
-                "sha256:a776bae1629c8d7198396fd93ec0265f8dd2341c553dc32b976168aaf0e6a636",
-                "sha256:aa94d617a4cd4cdf4af9b5af65100c036bce22280ebb15d8b5262e8273ebc6ba",
-                "sha256:b17d83b3d1610e571fedac21b2eb36b816654d6f7496004d6a0d32f99d1d8120",
-                "sha256:d73e3a96c38173e0aa5646c31bf8473bc3564837977dd480f5cbeacf1d7ef3a3",
-                "sha256:d91bc9f535599bed58f6d2e21a2724cb0c3895bf41c6403fe881391d29096f1d",
-                "sha256:ef216d13ac8d24d9cd851776662f75f8d29c9f2d05cdcc2d34a18d32463a9b0b",
-                "sha256:f6a5a85beb33e57998dc605b9dbe7deaa806385fdf5c4810fb849fcd04640c81",
-                "sha256:f92556f94e476c1b616e6daec5f7ddded2c082efa7cee7f31c7aeda615906ed8"
+                "sha256:0a817b961b46894c5ca8a66b599c745b9a3d9f822725221f0e0fe49dc043a3a3",
```
Comment (lsmith77) [inline Pipfile.lock]: did you need to update a dependency? I don't see any changes to the Pipfile itself.

Comment (Szarad) [reply Pipfile.lock]: Ah in the end no, I will rebase it

**Pull request merged**